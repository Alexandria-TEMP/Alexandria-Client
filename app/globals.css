@tailwind base;
@tailwind components;
@tailwind utilities;

@layer utilities {
  .text-balance {
    text-wrap: balance;
  }
}

@layer base {
  h1 {
    @apply text-2xl font-bold;
  }
  h2 {
    @apply text-xl font-semibold;
  }
  h3 {
    @apply text-lg;
  }
  h6 {
    @apply text-sm;
  }
}
