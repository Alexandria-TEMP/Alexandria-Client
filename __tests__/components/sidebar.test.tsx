import Sidebar from "@/components/sidebar";
import { expect, describe, it } from "@jest/globals";
import { render } from "@testing-library/react";

describe("Sidebar", () => {
  it("matches snapshot", () => {
    const { container } = render(
      <Sidebar
        title="Test sidebar"
        items={[
          { title: "First item", node: <p>First item</p> },
          { title: "Second item", node: <h2>Second item</h2> },
          { title: "Third item", node: <h4>One more</h4> },
        ]}
      />,
    );
    expect(container).toMatchSnapshot();
  });
});
