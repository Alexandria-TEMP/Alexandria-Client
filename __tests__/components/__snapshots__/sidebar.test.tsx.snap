// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarTest matches snapshot 1`] = `
<div>
  <div
    class="sticky top-20"
  >
    <h2>
      Test sidebar
    </h2>
    <h3>
      First item
    </h3>
    <p>
      First item
    </p>
    <div
      class="h-4"
    />
    <h3>
      Second item
    </h3>
    <h2>
      Second item
    </h2>
    <div
      class="h-4"
    />
    <h3>
      Third item
    </h3>
    <h4>
      One more
    </h4>
    <div
      class="h-4"
    />
  </div>
</div>
`;
